mineinabyss-systems/src/main/kotlin/com/mineinabyss/pvp/survival/SurvivalPvpCommandExecutor.kt

```diff
@@ -0,0 +1,34 @@
+package com.mineinabyss.pvp.survival
+
+import com.mineinabyss.components.playerData
+import com.mineinabyss.idofront.commands.execution.ExperimentalCommandDSL
+import com.mineinabyss.idofront.commands.execution.IdofrontCommandExecutor
+import com.mineinabyss.idofront.commands.extensions.actions.playerAction
+import com.mineinabyss.idofront.messaging.error
+import com.mineinabyss.idofront.messaging.success
+import com.mineinabyss.mineinabyss.core.layer
+import com.mineinabyss.mineinabyss.core.mineInAbyss
+
+@ExperimentalCommandDSL
+class SurvivalPvpCommandExecutor : IdofrontCommandExecutor() {
```

Comment:
We'll likely use a service/impl model for future adventure/survival features. Not a problem for this specifically but a note for me to set up an example so you guys can follow it in the future